stories/5-components/Forms/AuCheckboxGroup.stories.js

```diff
@@ -0,0 +1,52 @@
+import { hbs } from 'ember-cli-htmlbars';
+
+export default {
+  title: 'Components/Forms/AuCheckboxGroup',
+  argTypes: {
+    alignment: {
+      control: 'select',
+      options: ['default', 'inline'],
+      description: 'Choose the layout of the group.',
+    },
+    value: {
+      control: 'array',
+      description:
+        'Used to identify which checkboxes in the group are selected.',
+    },
+    disabled: {
+      control: 'boolean',
+      description: 'Adds a disabled state to all the checkboxes.',
+    },
+    onChange: {
+      control: 'function',
```

Comment:
Storybook has an actions system that can be used to log messages in the docs. I think we do it in a couple of places already.

```suggestion
      action: 'change', // or whatever name you choose
```